recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+{% set python_min = "3.9" %}
```

Comment:
This is the default and thus should be omitted.
```suggestion
```

Reply:
Modified. Thank you.